internal/integration/mysql_test.go

```diff
@@ -637,7 +647,14 @@ create table atlas_types_sanity
 					Name: "tTimestamp",
 					Type: &schema.ColumnType{Type: &schema.TimeType{T: "timestamp"},
 						Raw: "timestamp", Null: true},
-					Default: &schema.RawExpr{X: "CURRENT_TIMESTAMP"},
+					Default: &schema.RawExpr{
+						X: func() string {
+							if t.mariadb() {
+								return "current_timestamp()"
+							}
+							return "CURRENT_TIMESTAMP"
+						}(),
+					},
```

Comment:
why not t.valueByVersion(?

Reply:
because I have 3 versions, and the map feels too big.